project_utils/cpp/generation/SysIdSetup.cpp

```diff
@@ -133,6 +142,14 @@ void SetupEncoders(std::string encoderType, bool isEncoding, int period,
                    ->GetSelectedSensorVelocity() /
                cpr / 0.1;  // Conversion factor from 100 ms to seconds
       };
+    } else if (controllerName == "Venom") {
+      position = [=] {
+        return dynamic_cast<frc::CANVenom*>(controller)->GetPosition();
+      };
+      rate = [=] {
+        return dynamic_cast<frc::CANVenom*>(controller)->GetSpeed() /
+               60;  // Needs to be converted to rotations per second
```

Comment:
We should probably include the fact it's converting from RPM to rotations per second.